<launch>
    <rosparam command="load" file="$(find fastlio2_sam_lc)/config/localize.yaml" />
    <node pkg="fastlio2_sam_lc" type="localizer_node" name="localizer_node" output="screen"/>
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find fastlio2_sam_lc)/rviz/localize.rviz" />
</launch>